src/app/components/Embeds/OEmbed/index.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import OEmbedLoader from '.';
+
+export default {
+  title: 'Components/Embed Optimo',
+  component: OEmbedLoader,
+};
+
+const riddleBlock = {
```

Comment:
Just a small nitpick - is it worth importing this block from the Article fixture data - to avoid having multiple examples of the same riddle HTML in the repo?